test/extended/openstack/loadbalancers.go

```diff
@@ -270,6 +285,9 @@ var _ = g.Describe("[sig-installer][Suite:openshift/openstack][lb] The Openstack
 			o.Expect(err).NotTo(o.HaveOccurred())
 			o.Expect(strings.ToLower(pool.LBMethod)).Should(o.Equal(lbMethod), "Unexpected LBMethod on Openstack Pool: %q", pool.LBMethod)
 
+			// o.Expect(waitUntilNmembersReady(loadBalancerClient, pool, int(replicas), "ONLINE")).NotTo(o.HaveOccurred(),
```

Comment:
Do we need to report an issue with Vexxhost about LB not going into ONLINE status?

Reply:
Still not clear if it's vexxhost or octavia issue. I asked on #IRC channel but it looks it needs more analysis that should be part of OSP17.1 work (afaiu from @dulek).

I think we can go with the approach I implemented in my last change for the moment. I tested around 50 times in vexxhost and D/S setups and the tests are stable.

Reply:
Cool, happy to merge once the tests are successful.